cellprofiler/gui/help/search.py

```diff
@@ -274,9 +271,6 @@ def search_module_help(text):
     for title, help_text, pairs in matching_help:
         top += u"""<li><a href="#match{:d}">{}</a></li>\n""".format(match_num, title)
 
-        if help_text.find("<h1>") == -1:
```

Comment:
I thought this was unnecessary, but it turns out some pages have titles that don't exist in the body (hence this _is_ needed). The reason it wasn't matching properly was because some pages also have `<h1 class="title>`.